control_panel_api/tests/test_models.py

```diff
@@ -0,0 +1,89 @@
+from django.db.utils import IntegrityError
+from django.test import TestCase, TransactionTestCase
+
+from control_panel_api.models import (
+    Role,
+    Team,
+    TeamMembership,
+    User,
+)
+
+
+class MembershipsTestCase(TestCase):
+
+    @classmethod
+    def setUpTestData(cls):
+        # Users
+        cls.user_alice, _ = User.objects.get_or_create(username="Alice")
+        cls.user_bob, _ = User.objects.get_or_create(username="Bob")
+        cls.user_other, _ = User.objects.get_or_create(username="Other")
+        # Teams
+        cls.team_justice, _ = Team.objects.get_or_create(name="Justice Team")
+        cls.team_other, _ = Team.objects.get_or_create(name="Other Team")
+        # Roles
+        cls.role_maintainer, _ = Role.objects.get_or_create(
+            code="maintainer",
+            name="Maintainer",
+        )
+        cls.role_member, _ = Role.objects.get_or_create(
+            code="member",
+            name="Member",
+        )
+
+        # Add Alice to Justice team as maintainer
+        TeamMembership.objects.create(
+            team=cls.team_justice,
+            user=cls.user_alice,
+            role=cls.role_maintainer,
+        )
```

Comment:
This is a bit verbose - maybe in this test module there should be a function `add_user_to_team(user, team, role="Member")` or maybe a method `User.add_to_team(team, role)` or `Team.add_user(user, role)` or similar?

Reply:
Yes, we may want to add some methods like that in the future.